docs/user/bots.rst

```diff
@@ -2908,6 +2908,27 @@ writes it to `source.fqdn` or `destination.fqdn` if it is a hostname, or
 
 * `overwrite`: boolean, replace existing FQDN / IP address?
 
+uWhoisd
+^^^^^^^
+
+[uWhoisd](https://github.com/Lookyloo/uwhoisd) is a universal Whois server that supports
+caching and stores whois entries for historical purposes.
+
+The bot sends a request for `source.url`, `source.asn`, `source.fqdn`, or `source.ip`
```

Comment:
```suggestion
The bot sends a request for `source.url`, `source.asn`, `source.fqdn` or `source.ip` (the first existing field takes precedence)
```
or something like that

Reply:
I'll reorder them for now: url -> fqdn -> ip -> asn: it is closer to what I would expect in term of getting the most relevant whois entry for an intelmq event